modules/ROOT/pages/tools/cypher-shell.adoc

```diff
@@ -503,4 +503,10 @@ The following keyboard commands are available in interactive mode.
 |↹ (tab)
 |Autocompletion of commands and Cypher syntax.
 Suggestions for Cypher syntax is not complete.
+
+|`Home`
```

Comment:
I wonder what the Home and the End keys on Mac are. Perhaps, a combination of keys.
```suggestion
|Home (key)
```

Reply:
We use backticks only for code snippets.